### ISSUE TITLE
insufficient doc for PlayAreaNode and ControlAreaNode

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#469]
This identical doc appears in PlayAreaNode and ControlAreaNode:

```js
/**
 *
 * A node that creates a "Play Area" accessible section in the PDOM.
 * @author Michael Kauzmann (PhET Interactive Simulations)
 */
```

Problems:

(1) Copy-paste error in ControlAreaNode.

(2) This doc provides no meaningful information.  I.e. what is "a Play Area accessible section", what are its responsibilities, etc.

**Comment by pixelzoom** [comment]
Btw... I ended up here because instances of these nodes were added (also without documentation) to Molarity, a sim for which I'm the responsible developer.

**Comment by zepumph** [comment]
@pixelzoom how is that?

**Comment by zepumph** [comment]
Note work recently done in https://github.com/phetsims/scenery-phet/issues/381 means that these nodes are now created in ScreenView instead of directly in molarity (et all).

**Comment by pixelzoom** [comment]
👍 closing.
